test/configCases/target/umd-auxiliary-comments-string/index.js

```diff
@@ -0,0 +1,10 @@
+it("should run", function() {
+
+});
+
+it("should have auxiliary comment string", function() {
+	var fs = require("fs");
+	var source = fs.readFileSync(__filename, "utf-8");
+
+	source.should.containEql("//test comment");
```

Comment:
could you change this to `source.should.containEql("//test " + "comment");` elsewise it can find it's own source. Same for the other test case